src/App.tsx

```diff
@@ -1,61 +1,119 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './components/TodoList';
+
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
 
 export const App = () => {
+  const [addedTodos, setAddedTodos] = useState<Todo[]>(todos);
+  const [newTodoTitle, setNewTodoTitle] = useState<string>('');
+  const [selectedUserId, setSelectedUserId] = useState<number>(0);
+  const [titleError, setTitleError] = useState<string>('');
+  const [userError, setUserError] = useState<string>('');
```

Comment:
No need to explicitly define types where it can be implied.
Instead of `const [newTodoTitle, setNewTodoTitle] = useState<string>('');` simply use `const [newTodoTitle, setNewTodoTitle] = useState('');`; type `string` in implicit